common/trinary/tests/test_trit_tryte.c

```diff
@@ -22,7 +22,7 @@ void test_tryte_to_trit(void) {
   TEST_ASSERT_EQUAL_MEMORY(exp, trits, sizeof(exp));
 }
 
-void test_trit_to_tryte(void) {
+void test_tryte_to_trit(void) {
```

Comment:
And this one `test_trytes_to_trits` ?

Reply:
Also, for some reason this function is not actually run, could you add a `RUN_TEST` ?